packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
+Feature: Tax Document Submitted View
+        View that informs the user of their Tax Document submission status.
+
+	Background: A user has submitted their Tax Document
+		Given a user is on the Tax Document Submitted view
+
+	@motivating
+	Scenario: View Tax Document Submitted
+		Given the user has submitted a tax document
+		When they view Document Submitted
+		Then they should see the status of their Tax Document and Banking Information
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Form header is controlled via <documentType> prop
+	Given the <documentType>
+	Then the Tax Form header will display "<documentType> Tax Form"
+	Examples:
+		| documentType |
+		| W9 		   |
+		| W8-BEN       |
+		| W8-BEN-E     |
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Document Submitted badge view is controlled via props
```

Comment:
A good scenario title encompasses the most important part of the spec in one concise sentence

I would say something a bit more user focused is better. Maybe like "The date submitted and status of a tax form is displayed to end users"

"The Tax Document Submitted badge view is controlled via props" isn't wrong, but I feel like the core of the spec is that a user is able to see the status of date submitted of their tax form

Reply:
This feedback applies to more than just this spec